modules/datadog-configuration/modules/datadog_keys/versions.tf

```diff
@@ -0,0 +1,10 @@
+terraform {
```

Comment:
this file shouldn't be added as the provider lock provided by the calling component. 
e.g.
[datadog-monitors/versions.tf](https://github.com/cloudposse/terraform-aws-components/blob/master/modules/datadog-monitor/versions.tf)
and [it's calling file to this module](https://github.com/cloudposse/terraform-aws-components/blob/master/modules/datadog-monitor/provider-datadog.tf#L1-L5)

Reply:
we don't add versions here because then we'd have to keep them in sync rather than letting the calling module handle it

Reply:
but these modules can potentially be used independently, right? I understand not including a lockfile here or a `provider "aws" {}` but if there are resources in this submodule I think it is better to be explicit about the required version of provider, at least setting a minimum compatible version. it isn't required that it stays in sync with the parent module, as long as the version range is permissive.'